cabal-doctest.cabal

```diff
@@ -33,33 +33,30 @@ tested-with:
   GHC == 8.4.4
   GHC == 8.2.2
   GHC == 8.0.2
-  -- 2023-10-14: Drop CI for GHC 7.x
-  -- GHC == 7.10.3
-  -- GHC == 7.8.4
-  -- GHC == 7.6.3
-  -- GHC == 7.4.2
-  -- GHC == 7.2.2
-  -- GHC == 7.0.4
+  -- 2023-10-14: Dropped CI support for GHC 7.x
 
 source-repository head
   type:     git
-  location: https://github.com/haskellari/cabal-doctest
+  location: https://github.com/ulidtko/cabal-doctest
 
 library
   exposed-modules:  Distribution.Extra.Doctest
   other-modules:
   other-extensions:
   build-depends:
-    -- upper-bounds are major-major only, unless there is evidence for build failures
+    -- NOTE: contrary to PVP, some upper-bounds are intentionally set to major-major.
+    -- This is to increase signal-to-noise ratio of CI failures. "Too tight bounds"
+    -- is an extremely boring (and practically guaranteed, repeatedly) failure mode.
+    -- OTOH, genuine build failures due to breaking changes in dependencies are:
+    --  1) unlikely to occur, as this package is so small, moreso regularly;
+    --  2) best caught in CI pipelines that don't induce alert fatigue.
+    -- In any case, revisions may set tighter bounds afterwards, if exceptional
+    -- circumstances would warrant that.
       base       >=4.3  && <5
```

Comment:
Shouldn't you raise to `base >= 4.9` (dropping GHC 7)?

Reply:
Good catch, thanks... fixed.